### PR TITLE
Correct errors in documentation

### PR DESCRIPTION
**Pull request by tlevine**
## PR Summary
I corrected three errors in documentation: two missing periods, and one misspelled word.

### CODE REVIEW

File: doc/api/api_changes.rst [hunk 1]
```diff
@@ -39,7 +39,7 @@ API Changes for 3.0.0
 Drop support for python 2
 -------------------------
 
-Matplotlib 3 only supports python 3.5 and higher
+Matplotlib 3 only supports python 3.5 and higher.
```
Comment (story645) [inline doc/api/api_changes.rst]: added on some minor grammar changes to your PR, but I think this sentence doesn't need the . (and Python should be capitalized).

File: doc/api/animation_api.rst [hunk 2]
```diff
@@ -60,8 +60,8 @@ general gist is to take an existing bit map (in our case a mostly
 rasterized figure) and then 'blit' one more artist on top.  Thus, by
 managing a saved 'clean' bitmap, we can only re-draw the few artists
 that are changing at each frame and possibly save significant amounts of
-time.  When using blitting (by passing ``blit=True``) the core loop of
-`FuncAnimation` gets a bit more complicated ::
+time.  When using blitting (by passing ``blit=True``), the core loop of
+`FuncAnimation` gets a bit more complicated. ::
```
Comment (jklymak) [inline doc/api/animation_api.rst]: This should be a colon, if anything here.

Comment (story645) [reply doc/api/animation_api.rst]: what should be a colon?

Comment (jklymak) [reply doc/api/animation_api.rst]: The period.  The sentence is implicitly carried on in the code below.  I'd actually just leave w/o the period or colon, but a period is incorrect.

Comment (tacaswell) [reply doc/api/animation_api.rst]: Please refrain from doing copy-editting via github comment.

Comment (jklymak) [reply doc/api/animation_api.rst]: ... its a copy-edit PR.  How would you like the copy editing corrected?  Just force-push?

**Pull request merged**